app/views/favorites/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Your Favorite Pets</h1>
+<section>
+  <% if favorite.total_count > 0 %>
+    <% @favorited.each do |pet| %>
+      <div id="favorite-<%= pet.id %>" class="pet_block">
```

Comment:
There was an accidental style tag that was here used in discussion... If your version has a style="display:flex" this is where it came from, and it has been removed...